configs/setups/awicm3/awicm3.yaml

```diff
@@ -1,196 +1,657 @@
+                                LECOMPGRID: "${lecompgrid}"
+                        NAMCLDP:
+                                SCLCT_SWITCH: "${sclct_switch}"
+                        NAMFPG:
+                                NFPMAX: "${res_number}"
+                        NAMFPD:
+                                NLAT: "${nlat}"
+                                NLON: "${nlon}"
+                        NAMPAR0:
+                                NSPECRESMIN: "${nspecresmin}"
+                        NAMDYNA:
+                                LGRADSP: "${lgradsp}"
+                        NAMNMI:
+                                LASSI: "${lassi}"
+                        NAMCT0:
+                                LFDBOP: "${lfdbop}"
+                                LSMSSIG: "${lsmssig}"
+                        # Where the hell do these namelist changes come from? I don't even remember writting them down... and they are not part of the original fort.4???
+                        #NAMGRIB:
+                        #        NJTKDATE: "${start_date}"
+                        #        NJTKOFFSETSTEP: "${steps_since_initial}"
+                        NAMORB:
+                                LCORBMD:  "${orb_switch}"
+                                ORBMODE: "${orb_mode}"
+                                ORBIY: "${orb_iyear}"
+        # Namelist changes for wam_namelist not included yet as the first test is for DRH6
+        # that has no wave model. TODO: implement in the near future
+
+        runtime_environment_changes:
+                add_export_vars:
+                        # Turn on FESOM coupling
+                        - 'FESOM_USE_CPLNG="active"'
+                        - 'ECE_CPL_NEMO_LIM="false"'
+                        - 'ECE_CPL_FESOM_FESIM="true"'
+                        - 'ECE_AWI_CPL_FESOM="true"'
+
+        res_number_tl: ${resolution}${levels} # @JOAKIM: what is this variable for? @JAN: Is the directory correct?
+        choose_general.resolution:
+                TL159_CORE2:
+                        nproca: 279 # @JAN, are these numbers correct for this resolution?
+                        nprocb: 1
+
+        # Namelist modifications for truncation types
+        nlon: "$(( ${nlat}*2 ))"
+        nspecresmin: "$(( ${res_number}+1 ))"
+        choose_truncation:
+                TCO:
+                        nlat: "$(( (${res_number}+1)*2 ))"
+                        lecompgrid: true
+                        lgradsp: false
+                        lassi: false
+                        lfdbop: false
+                        lsmssig: false
+                TL:
+                        nlat: "$(( ${res_number}+1 ))"
+                        lecompgrid: false
+                        lgradsp: true
+                        lassi: true
+                        lfdbop: true
+                        lsmssig: true
+
+        #@JAN/JOAKIM: is ok to ignore this files? anything else that needs to be removed?
```

Comment:
I'm not sure what this does. Cleanup after the run is done to save space? Maybe this is obsolete, since we link these from pool dir now?

Reply:
No idea what these things are. Something about modifying the LSM? 
Yes, we link everything we need from the pool dirs, e.g. 159_4, ifsdata, etc.

Reply:
To be clear, the comment by @mandresm  is about the l239 and below. It a bit strange how the github review comment shows only the lines above. l235 and l236 are dealing with TL vs TCO differences.

Reply:
This comment refers to lines 241-243:
```
        remove_input_sources:
                - rtables
                - vtables
```
It's an old comment I should have removed cause it is a correct action: as you know, ESM-Tools first reads `oifs.yaml` and then `awicm3.yaml`. In `oifs.yaml` the rtables and vtables are copied into the experiment directory (specified in `input_sources` list). However, for AWICM3 we don't want any of this copying, at least at the time I was working on that part, as the rtables origin paths are specified in the namelist. That is why I chose to remove the sources so that copying for AWICM3 does not happen. I think in reality we want to do the copying into the experiment folder, and then do the linking for each run folder but this is something I did not have time to address yet (I have a note about it somewhere from a talk with @JanStreffing ).

Reply:
The last sentence somehow contradicts the one before? I agree with the one before and think vtables and rtables don't need to be copied them at all. I don't think they ever change. I wouldn't even know how to or what format they are in. Its not netcdf or grib or txt.

Reply:
Ok, I was not sure if they needed to be copied or not (I didn't remember). I knew that it was currently working as it was, so I wanted to look at my notes to confirm, but thanks for the clarification. They will never be copied :)